src/appmixer/intelswift/core/NewContact/NewContact.js

```
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+
+    async start(context) {
+
+        const tenantId = context.profileInfo.tenantId;
+        context.log({ stage: 'profile info', info: context, tenantId, check: "1" });
```

ESLint

[error] Strings must use singlequote.

(quotes)